src/test/java/org/apache/commons/imaging/formats/png/PngReadTest.java

```diff
@@ -74,4 +78,22 @@ public void test() throws Exception {
         }
     }
 
+    /**
+     * If the PNG image data contains an invalid ICC Profile, previous versions would
+     * simply rethrow the IAE. This test verifies we are instead raising the documented
+     * {@literal ImageReadException}.
+     *
+     * <p>See Google OSS Fuzz issue 33691</p>
+     *
+     * @throws IOException if it fails to read the test image
+     */
+    @Test
+    public void testUncaughtExceptionOssFuzz33691() throws IOException {
+        final String input = "/images/png/oss-fuzz-33691/clusterfuzz-testcase-minimized-ImagingPngFuzzer-6177282101215232";
+        final String file = PngReadTest.class.getResource(input).getFile();
+        final PngImageParser parser = new PngImageParser();
+        assertThrows(ImageReadException.class, () -> {
+            parser.getBufferedImage(new ByteSourceFile(new File(file)), Collections.emptyMap());
```

Comment:
I think you can use lambda expression.
`        assertThrows(ImageReadException.class, () -> parser.getBufferedImage(new ByteSourceFile(new File(file)), Collections.emptyMap()));
`
But in general LGTM

Reply:
Sounds good to me @arturobernalg , if you add a [suggestion](https://til.hashrocket.com/posts/szgdi4at59-github-insert-a-suggestion-) here I'll merge it (otherwise will update the PR tomorrow :bed: :crescent_moon: )

Reply:
HI @kinow 
Good night
```suggestion
         assertThrows(ImageReadException.class, () -> parser.getBufferedImage(new ByteSourceFile(new File(file)), Collections.emptyMap()));
```